hyperdom.js

```diff
@@ -0,0 +1,39 @@
+var Mount = require('./mount');
+var hyperdom = require('hyperdom');
+var createMonkey = require('./create');
+var window = require('global');
+var createTestDiv = require('./createTestDiv')
+var setTestUrl = require('./setTestUrl')
+
+module.exports = function(app, options) {
+  return new Mount(app, {
+    stopApp: function(){
+    },
+    startApp: function(){
+      if (options && options.router) {
+        options.router.reset()
+      }
+      var app = this.app;
+
+      if (Mount.runningInNode) {
+        try {
+          var vquery = require('vdom-query');
+        } catch (e) {
+          throw new Error('you must `npm install vdom-query --save-dev` to run tests in node');
+        }
+        var vdom = hyperdom.html('body');
+
+        var monkey = createMonkey(vdom);
+        monkey.set({$: vquery, visibleOnly: false, document: {}});
+
+        hyperdom.appendVDom(vdom, app, Object.assign({ requestRender: setTimeout, window: window }, options));
```

Comment:
good catch... right, i have to confess to my current thinking on this subject: we should use standard modern javascript methods where available and recommend users to polyfill on browsers that don't have them. We shouldn't use `lowscore` or `underscore` for methods that are now included in JavaScript, firstly because we always forget to use the non-standard `underscore` version, and because it keeps us dependent on `underscore` (or `lowscore`).

Think that'll work?